.storybook/webpack.config.js

```diff
@@ -1,6 +1,23 @@
 const path = require("path");
 
-module.exports = ({ config }) => {
+module.exports = ({ config }) => {  
+  // modify storybook's file-loader rule to avoid conflicts with our inline svg
+  const fileLoaderRule = config.module.rules.find(rule => rule.test.test('.svg'));
+  fileLoaderRule.exclude = /\.svg$/;
+  
+  config.module.rules.push({
+    test: /\.svg$/,
+    exclude: /node_modules/,
+    use: [
+      {
+        loader: require.resolve("babel-loader")
+      },
+      {
+        loader: require.resolve("react-svg-loader")
```

Comment:
How does it compare to `@svgr/webpack` ?

Reply:
`@svgr/webpack` has rollup port but it's buggy. `react-svg-loader` is a nice alternative